QgisModelBaker/gui/validate.py

```diff
@@ -521,6 +530,15 @@ def _open_form(self, t_ili_tid):
             self.iface.layerTreeView().setCurrentLayer(layer)
             self.iface.openFeatureForm(layer, feature, True)
 
+    def _select_feature_in_attributetable(self, t_ili_tid):
+        layer, feature = self._get_feature_in_project(t_ili_tid)
+        if layer and feature:
+            self.iface.layerTreeView().setCurrentLayer(layer)
+            layer.removeSelection()
+            layer.select(feature.id())
+            attribute_table = self.iface.showAttributeTable(layer)
+            attribute_table.findChild(QAction, "mActionSelectedFilter").trigger()
```

Comment:
It would be good to check if the returned widget is valid to avoid crashes if something changes upstream. For example mActionSelectedFilter renamed or so

Reply:
good input